index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="navbar">
+        <ul>
+             <li><a href="#about">About</a></li>
+             <li><a href="#projects">Work</a></li>
+             <li><a href="#contact">Contact</a></li>
+            
+        </ul>
+    </div>
```

Comment:
The idea of the project is to give comments about the usage of HTML semantic tags and HTML and CSS best practices you can also comment on the usage of `navbar` since the user doesn't use any sematic tag it should be wrapped inside the `header` tag we can use the `nav` tag to wrap the `ul`. Please add this before requesting the next review.